rocksdb_replicator/replicated_db.cpp

```diff
@@ -78,7 +79,7 @@ namespace replicator {
 rocksdb::Status RocksDBReplicator::ReplicatedDB::Write(
     const rocksdb::WriteOptions& options,
     rocksdb::WriteBatch* updates,
-    const common::Config& config,
+    const common::Config& , // ignore this value
```

Comment:
should we remove this after the new change is live? maybe a TODO

Reply:
Yes We need to do that , but will have that as a separate change ...

Reply:
can you add a TODO to make this clear?

Reply:
sure will do ..